web/wp-content/mu-plugins/pantheon/pantheon-login-form-mods.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Modify the WordPress login form for Pantheon
+ */
+
+ /**
+  * Should we proceed with adding the
+  * return to Pantheon button?
+  *
+  * Only if:
+  * We are on a Pantheon subdomain and
+  * "RETURN_TO_PANTHEON_BUTTON" is not false
+  */
+$show_return_to_pantheon_button = apply_filters( 'show_return_to_pantheon_button', ( 
+    ( ! defined('RETURN_TO_PANTHEON_BUTTON') || RETURN_TO_PANTHEON_BUTTON ) &&
+    (
+        false !== stripos( get_site_url(), 'pantheonsite.io') ||
+        ( isset( $_SERVER['HTTP_HOST'] ) && false !== stripos( $_SERVER['HTTP_HOST'], 'pantheonsite.io') )
+     )
+) );
```

Comment:
:no_entry_sign: **Error**: Closing parenthesis of a multi-line function call must be on a line by itself (*PEAR.Functions.FunctionCallSignature.CloseBracketLine*).